src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv.sarif

```diff
@@ -0,0 +1,1353 @@
+                "region": {
+                  "startLine": 1842,
+                  "startColumn": 24,
+                  "endLine": 1842,
+                  "endColumn": 28
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "note",
+          "message": "Method FileFilterUtils.asFileFilter(..) could be confused with overloaded method [[\"asFileFilter\"|\"relative://src/main/java/org/apache/commons/io/filefilter/FileFilterUtils.java:515:18000:12\"]], since dispatch depends on static types.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/filefilter/FileFilterUtils.java",
+                "region": {
+                  "startLine": 503,
+                  "startColumn": 32,
+                  "endLine": 503,
+                  "endColumn": 43
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Class response too high (430). It is a warning for the value to be higher or equal to 350.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/FileUtils.java",
+                "region": {
+                  "startLine": 79,
+                  "startColumn": 14,
+                  "endLine": 79,
+                  "endColumn": 22
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Type specialization index too high (5). It is a warning for the value to be higher or equal to 4.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/output/TeeOutputStream.java",
+                "region": {
+                  "startLine": 29,
+                  "startColumn": 14,
+                  "endLine": 29,
+                  "endColumn": 28
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Type specialization index too high (4). It is a warning for the value to be higher or equal to 4.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/input/TeeInputStream.java",
+                "region": {
+                  "startLine": 40,
+                  "startColumn": 14,
+                  "endLine": 40,
+                  "endColumn": 27
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Lines of code in files too high (1168). It is a warning for the value to be higher or equal to 1000.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/FileUtils.java",
```

Comment:
>src/main/java/org/apache/commons/io/FileUtils.jav [](start = 24, length = 49)

I think these URIs should be prefixed with a slash. 

See section 4.2
[http://www.ietf.org/rfc/rfc3986.txt](http://www.ietf.org/rfc/rfc3986.txt)